app/lib/actions/katello/repository/multi_clone_contents.rb

```diff
@@ -18,34 +20,29 @@ def plan(extended_repo_mapping, options)
 
             concurrence do
               extended_repo_mapping.each do |source_repos, dest_repo_map|
+                dest_repo_map[:matching_content] = check_matching_content(dest_repo_map[:dest_repo], source_repos)
```

Comment:
`dest_repo_map[:matching_content]` is always false for me in this scenario:
1) Sync the `needed_errata` repo
2) Add it to the content view
3) Publish n times

I'm assuming that indexing shouldn't be happening after the first CV publish.  I noticed that `dest_repo_map[:dest_repo]`, which is fed to `check_matching_content`, is archived and has no RPMs.  Seems like is is why my indexing always occurs.

Reply:
It appears that on a subsquent publish, the index content action is invoked because the sub-action, CheckMatchingContent returns the output of "matching_content" as false, which is confusing since it's the same repository.

Reply:
I believe I was referring to the index action being planned.  I remember seeing the index action happening more times than I expected. If you're not seeing this then I might have to go back and try again, it's been a while.

Reply:
Justin pointed out that these changes included a boolean that is actually a deferred value not set until the run phase of the action, so it is always truthy. This is probably part of the issue! Hold off on any retesting until I can refactor.